editor/sidebar/last-revision/index.js

```diff
@@ -81,6 +81,10 @@ class LastRevision extends Component {
 		const { revisions } = this.state;
 		const lastRevision = revisions.length ? revisions[ 0 ] : null;
```

Comment:
Good suggestion. Was trying to minimize the change set.  I will take another look.